addons/binding/org.openhab.binding.somfytahoma/ESH-INF/thing/gateway.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="somfytahoma" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="gateway">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		<label>Somfy Tahoma Gateway</label>
+		<channels>
+			<channel id="version" typeId="version"></channel>
+			<channel id="status" typeId="status"></channel>
+		</channels>
+		<config-description-ref uri="thing-type:somfytahoma:gateway"/>
+	</thing-type>
+
+	<!-- Gateway firmware version -->
+	<channel-type id="version">
```

Comment:
for me it is a channel, because I'm getting notification when device updates (rule)

Reply:
Since this is more likely a geek channel, I've decided to remove the version channel and I've introduced a ```Thing.PROPERTY_FIRMWARE_VERSION``` property.